samcli/lib/list/stack_outputs/stack_output_to_table_mapper.py

```diff
@@ -0,0 +1,16 @@
+"""
+Implementation of the stack output to table mapper
+"""
+from samcli.lib.list.list_interfaces import Mapper
+from samcli.views.concrete_views.rich_table import RichTable
+
+
+class StackOutputToTableMapper(Mapper):
+    def map(self, data: list) -> RichTable:
+        output = RichTable(title="Stack Outputs", table_options={"show_lines": True})
```

Comment:
Has this passed UX reviews? are we looking at using the new UX powers that rich library offers us to change the output style of deploy?

Reply:
Changed the table used for table format. Have meeting with Molly regarding UX details this Friday